src/widgets/auth/views/loginWithPasswordViewComponent.jsx

```diff
@@ -30,17 +29,16 @@ export const LoginWithPasswordForm = createForm({
                 identifierField({
                     key: 'identifier',
                     defaultValue: username,
+                    withPhoneNumber: true,
                     readOnly: true,
                 }, config)
                 :
-                simpleField({
-                    key: 'email',
-                    label: 'email',
-                    type: 'email',
-                    autoComplete: 'email',
```

Comment:
Use the identifierField as the identifier used for login can either be an email or a customIdentifier.